src/grid/grid.component.ts

```diff
@@ -838,7 +842,7 @@ export class IgxGridComponent implements OnInit, OnDestroy, AfterContentInit, Af
                 maxSummaryLength = currentLength;
             }
         });
-        return maxSummaryLength * (this.tfoot.nativeElement.clientHeight ? this.tfoot.nativeElement.clientHeight : 36);
+        return maxSummaryLength * (this.tfoot.nativeElement.clientHeight ? this.tfoot.nativeElement.clientHeight : 36.36);
```

Comment:
Make this `36.36` a `const` declaration somewhere in the component. Where is the number coming from anyways?

Reply:
We are going to change this behavior, although for now we are going to declare a const in the component.